hive/src/main/java/com/netease/arctic/hive/utils/UpgradeHiveTableUtil.java

```diff
@@ -136,6 +143,7 @@ private static void hiveDataMigration(
       }
     }
     HiveMetaSynchronizer.syncHiveDataToArctic(arcticTable, arcticHiveCatalog.getHMSClient());
+    fillPartitionProperties(arcticTable, arcticHiveCatalog, hiveTable);
```

Comment:
May need to reload `hiveTable` after `HiveMetaSynchronizer#syncHiveDataToArctic` is called.